_includes/pbc/all/integrate-features/202204.0/integrate-glue-api/integrate-the-cart-glue-api.md

```diff
@@ -180,6 +180,25 @@ class CartsRestApiConfig extends SprykerCartsRestApiConfig
 }
 ```
 
+### Configure quote creation
+
+If you want to create a cart for the newly authenticated customer while merging guest cart with customer cart, you can enable it by adjusting the configuration constant:
```

Comment:
@Zingeon  In the phrase "you can enable it", what do you mean by "it"?

Reply:
@VadymSachenko the following rephrasing sounds good? `You can enable creating a cart for the newly authenticated customer while merging guest cart with customer cart by adjusting the configuration constant:`